pkg/util/source/doc.go

```diff
@@ -0,0 +1,7 @@
+// TODO: this package locally re-implements the following upstream patch:
+//
+// https://github.com/kubernetes-sigs/controller-runtime/pull/383
+//
+// If upstream accepts the patch, this package can be deleted and replaced by
+// upstream code.
+package source
```

Comment:
Looks like your forgot to remove this file.

Reply:
I must have failed to isolate those changed in the deleted commit. Thanks!